src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckNullCaseLabel.java

```diff
@@ -15,6 +15,24 @@ public void switchWithNullLabelTest(SwitchInput i) {
             case SECOND: counter--;
             case null: counter=counter+2;
```

Comment:
So we had an existing case here.

@mahfouz72 Can you clarify why this is an issue if we had an existing case? I am not seeing any real differences between this and your next new input.

Reply:
His comment was in reference to no diffs in the report.

Reply:
> I am not seeing any real differences between this and your next new input.

in my new input the case null is grouped with other cases but in old input the case null is an alone case not grouped 


```Java
case SECOND:             // no statement here so both cases grouped under same CASR_GROUP
case null: counter=counter+2;
```

```Java
case SECOND: counter--; // each case is under its CASE_GROUP
case null: counter=counter+2; 
```

Reply:
Thanks, I see now. We were only check first item in group and in your example, we have multiple.

````
        |      |  |--CASE_GROUP -> CASE_GROUP [9:12]
        |      |  |  |--LITERAL_CASE -> case [9:12]
        |      |  |  |  |--EXPR -> EXPR [9:17]
        |      |  |  |  |  `--IDENT -> FIRST [9:17]
        |      |  |  |  `--COLON -> : [9:22]
        |      |  |  |--LITERAL_CASE -> case [10:12]
        |      |  |  |  |--EXPR -> EXPR [10:17]
        |      |  |  |  |  `--IDENT -> SECOND [10:17]
        |      |  |  |  `--COLON -> : [10:23]
        |      |  |  |--LITERAL_CASE -> case [11:12]
        |      |  |  |  |--EXPR -> EXPR [11:17]
        |      |  |  |  |  `--LITERAL_NULL -> null [11:17]
````